scripts/extensions/auto-tagging-widget/src/ImageTaggingComponent/ImageTaggingComponent.tsx

```diff
@@ -0,0 +1,370 @@
+                    res.push({
+                        title: concept.name,
+                        type: key,
+                        pubStatus: true,
+                        weight: 1,
+                    });
+                });
+            }
+        }
+        return res;
+    }
+
+    handleClickImage = (image: IImage) => {
+        this.setState({selectedImage: image});
+    }
+
+    render() {
+        const {style} = this.props;
+        const {isLoading, selectedImage, images} = this.state;
+
+        return (
+            <ToggleBox
+                className="toggle-box--circle"
+                title={
+                    isLoading
+                        ? gettext('loading image suggestions...')
+                        : gettext('image suggestions ({{n}})', {n: images.length})
+                }
+                initiallyOpen={true}
+                badge={(
+                    <IconButton
+                        id="image-suggestions-info-btn"
+                        icon="info-sign"
+                        size="small"
+                        ariaValue="info"
+                    />
+                )}
+            >
+                <Popover
+                    title={gettext('Information')}
+                    placement="bottom-end"
+                    triggerSelector="#image-suggestions-info-btn"
+                    zIndex={999}
+                >
+                    {gettext(
+                        'Image suggestions are based on generated tags'
+                        + ' and can be added to article content by dragging.',
+                    )}
+                </Popover>
+
+                <div style={style}>
+                    {(() => {
+                        if (isLoading) {
+                            return (
+                                <div
+                                    style={{
+                                        display: 'flex',
+                                        alignItems: 'center',
+                                    }}
+                                >
+                                    <div className="spinner-big" />
+                                </div>
+                            );
+                        } else if (images.length < 1) {
+                            return null;
+                        } else {
+                            return (
+                                <div style={gridContainerStyle}>
+                                    <div style={gridItemLeftStyle}>
+                                        <figure style={selectedCardStyle}>
+                                            <div style={{maxHeight: '72%'}}>
+                                                <img
+                                                    style={imageStyle}
+                                                    alt=""
+                                                    src={selectedImage?.imageUrl}
```

Comment:
`<img` shouldn't be rendered at all if we don't have a url. Same for figcaption.